sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -165,3 +165,23 @@ directive:
       transform: >
           $["version"] = "2022-04-01-preview";
 ```
+
+### Invalid urls
+
+### Mismatching task types
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationSummarizationTask"]
+      transform: >
+        $["x-ms-discriminator-value"] = "IssueResolutionSummarization";
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationPIITask"]
+      transform: >
+        $["x-ms-discriminator-value"] = "ConversationPII";
+```
```

Comment:
Tip: you can declutter like so:
```suggestion
```yaml
directive:
- from: swagger-document
  where: $["definitions"]["AnalyzeConversationSummarizationTask"]
  transform: >
    $["x-ms-discriminator-value"] = "IssueResolutionSummarization";

- from: swagger-document
  where: $["definitions"]["AnalyzeConversationPIITask"]
  transform: >
    $["x-ms-discriminator-value"] = "ConversationPII";
```

It's a YAML array so you can put it under one directive for brevity. You can also elide the whitespace on

Reply:
Separately, can we rename these using `x-ms-client-name` in the swagger so all autorest-generated clients have these changes? Or, at the very least, put the transforms in the readme.md in the swagger repo. It's best to avoid per-repo transforms if possible.

Reply:
yeah sure i'm working on refactoring the directives file now

Reply:
actually, after speaking with the service people, they told me that APIM/service changes will catch up to swagger definitions which means this entire section named `Fix Swagger/API mismatch errors` will be removed from swagger directives.